pkgs/applications/audio/audacious/default.nix

```diff
@@ -1,69 +1,79 @@
-{ stdenv, fetchurl, pkgconfig, glib, gtk3, libmowgli, libmcs
-, gettext, dbus_glib, libxml2, libmad, xorg, alsaLib, libogg
-, libvorbis, libcdio, libcddb, flac, ffmpeg, makeWrapper
-, mpg123, neon, faad2, gnome3
+{ stdenv, fetchurl, pkgconfig, autoconf, makeWrapper, gettext
+, glib, gtk3, libmowgli, libmcs, dbus_glib, libxml2, xorg, gnome3
+, alsaLib, libpulseaudio, libjack2, fluidsynth
+, libmad, libogg, libvorbis, libcdio082, libcddb, flac, ffmpeg, mpg123
+, libcue, libmms, libbs2b, libsndfile, libmodplug, libsamplerate
+, soxr, lirc, curl, wavpack, neon, faad2, lame, libnotify, libsidplayfp
 }:
 
-let version = "3.5.2"; in
-
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   name = "audacious-${version}";
+  version = "3.7.2";
 
   src = fetchurl {
-    url = "http://distfiles.audacious-media-player.org/audacious-${version}.tar.bz2";
-    sha256 = "0mhrdj76h0g6q197wgp8rxk6gqsrirrw49hfidcb5b7q5rlvj59r";
+    url = "http://distfiles.audacious-media-player.org/audacious-${version}-gtk3.tar.bz2";
+    sha256 = "1pvyxi8niy70nv13kc16g2vaywwahmg2650fa7v4rlbmykifk75z";
   };
 
   pluginsSrc = fetchurl {
-    url = "http://distfiles.audacious-media-player.org/audacious-plugins-${version}.tar.bz2";
-    sha256 = "1nacd8n46q3pqnwavq3i2ayls609gvxfcp3qqpcsfcdfz3bh15hp";
+    url = "http://distfiles.audacious-media-player.org/audacious-plugins-${version}-gtk3.tar.bz2";
+    sha256 = "0gxka0lp9a35k2xgq8bx69wyv83dvrqnpwcsqliy3h3yz6v1fv2v";
   };
 
-  buildInputs =
-    [ gettext pkgconfig glib gtk3 libmowgli libmcs libxml2 dbus_glib
-      libmad xorg.libXcomposite libogg libvorbis flac alsaLib libcdio
-      libcddb ffmpeg makeWrapper mpg123 neon faad2 gnome3.defaultIconTheme
-    ];
+  buildInputs = [
+    pkgconfig autoconf makeWrapper gettext glib gtk3 libmowgli
+    libmcs dbus_glib libxml2 xorg.libXcomposite gnome3.defaultIconTheme
+    alsaLib libjack2 libpulseaudio fluidsynth
+    libmad libogg libvorbis libcdio082 libcddb flac ffmpeg mpg123
+    libcue libmms libbs2b libsndfile libmodplug libsamplerate
+    soxr lirc curl wavpack neon faad2 lame libnotify libsidplayfp.out
+  ];
+
+  configureFlags = [ "--enable-statusicon" ];
 
-  # Here we build bouth audacious and audacious-plugins in one
+  # Here we build both audacious and audacious-plugins in one
   # derivations, since they really expect to be in the same prefix.
   # This is slighly tricky.
-  builder = builtins.toFile "builder.sh"
-    ''
-      # First build audacious.
-      (
-        source $stdenv/setup
-        genericBuild
-      )
+  builder = builtins.toFile "builder.sh" ''
```

Comment:
At some point this should be refactored. Maybe use two derivations for audacious and audacious-plugins? Fine to merge for now, though.